versions/3.0.md

```diff
@@ -389,16 +389,16 @@ All objects defined within the components object will have no effect on the API
-<a name="responsesDefinitionsObject"></a> | [Responses Definitions Object](#responsesDefinitionsObject) | Reusable responses objects.
-<a name="parametersDefinitionsObject"></a> | [Parameters Definitions Object](#parametersDefinitionsObject) | An object to hold parameters to be reused across operations. Parameter definitions can be referenced to the ones defined here.
-<a name="requestBodyDefinitionsObject"></a> | [Request Body Definitions Object](#requestBodyDefinitionsObject) | An object to hold request body definitions to be reused across operations. Request Body definitions can be referenced to the ones defined here.
-<a name="responseHeadersDefinitionsObject"></a> | [Response Headers Definitions Object](#responseHeadersDefinitionsObject) | Response headers to reuse across the specification.
-<a name="securityDefinitionsObject"></a> | [Security Definitions Object](#securityDefinitionsObject) | Security definitions to reuse across the specification.
-<a name="linksDefinitionObject"></a> | [Link Definitions Object](#linksDefinitionsObject) | Link definitions to reuse across the specification.
-<a name="callbacksDefinitionObject"></a> | [Callback Definitions Object](#callbacksDefinitionsObject) | Callback definitions to reuse across the specification.
+<a name="componentsDefinitionsObject"></a>definitions | [Definitions Object](#definitionsObject) | A hash containing payload definitions for the specification.
+<a name="componentsResponsesDefinitionsObject"></a>responsesDefinitions | [Responses Definitions Object](#responsesDefinitionsObject) | Reusable responses objects.
+<a name="componentsParametersDefinitionsObject"></a>parametersDefinitions | [Parameters Definitions Object](#parametersDefinitionsObject) | An object to hold parameters to be reused across operations. Parameter definitions can be referenced to the ones defined here.
+<a name="componentsRequestBodyDefinitionsObject"></a>requestBodyDefinitions | [Request Body Definitions Object](#requestBodyObject) | An object to hold request body definitions to be reused across operations. Request Body definitions can be referenced to the ones defined here.
+<a name="componentsResponseHeadersDefinitionsObject"></a>responseHeadersDefinitions | [Response Headers Definitions Object](#responseObject) | Response headers to reuse across the specification.
+<a name="componentsSecurityDefinitionsObject"></a>securityDefinitions | [Security Definitions Object](#securityDefinitionsObject) | Security definitions to reuse across the specification.
+<a name="componentsLinksDefinitionObject"></a>linkDefinitions | [Link Definitions Object](#linkObject) | Link definitions to reuse across the specification.
+<a name="componentsCallbacksDefinitionObject"></a>callbackDefinitions | [Callback Definitions Object](#callbackObject) | Callback definitions to reuse across the specification.
```

Comment:
Both `#callbackDefinitionsObject` and `#linkDefinitionsObject` are still missing. Or are they the same as the callback Object and links object linked here? Then the name of the links here should be changed, too.